### PR TITLE
rm Scotthelme.co.uk.xml

### PR DESCRIPTION
**Pull request by gloomy-ghost**
Preloaded

[Chromium](https://cs.chromium.org/chromium/src/net/http/transport_security_state_static.json?amp&sq=package:chromium&l=1200), [Firefox](https://dxr.mozilla.org/mozilla-aurora/source/security/manager/ssl/nsSTSPreloadList.inc#8823), [Tor](https://gitweb.torproject.org/tor-browser.git/tree/security/manager/ssl/nsSTSPreloadList.inc?h=tor-browser-45.4.0esr-6.5-1#n9747)

### CODE REVIEW

File: src/chrome/content/rules/Scotthelme.co.uk.xml [hunk 1]
```diff
@@ -1,14 +0,0 @@
-<!-- 
-    More rulesets for sites by Scott Helme:
-	    - Report-uri.io.xml
-	    - Securityheaders.io.xml
```
Comment (J0WI) [inline src/chrome/content/rules/Scotthelme.co.uk.xml]: Please also update the references.

**Comment by J0WI** [comment]
report-uri.io and securityheaders.io have a solid preloaded HSTS config, I think it's safe to remove them all.

**Comment by gloomy-ghost** [comment]
See #7134 #7135

**Comment by J0WI** [comment]
You should merge them into this PR. Otherwise this will result in merge conflicts.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**